aep/general/example.oas.yaml

```
@@ -411,6 +412,7 @@
                     'in': 'path',
                     'required': true,
                     'schema': { 'type': 'string' },
+                    'x-aep-resource-reference': { 'resource': 'Publisher' },
```

```suggestion
                    'x-aep-resource-reference': { 'resource': 'publisher' },
```

same with the others - we use lower kebab case.